### DISCUSSION TITLE
ADD CHANGE ICON AND VERSION/VERSION CODE AND SUPPORTE FOR CREATE NEW ACTIVITES

### DISCUSSION THREAD

**Discussion by walterappdev** [tyron12233/CodeAssist#243] [Ideas]

**Comment by walterappdev** [comment]
ADD CHANGE ICON AND VERSION/VERSION CODE AND SUPPORT FOR CREATE NEW ACTIVITES

**Comment by hassanmahfuj** [comment]
This is not Sketchware my boy. You have to do this by the Right Way.
